adapters/smaato/smaatotest/videosupplemental/bad-imp-ext-request.json

```diff
@@ -0,0 +1,292 @@
+{
+    "mockBidRequest": {
+        "id": "447a0a1d-389d-4730-a418-3777e95de7bd",
+        "imp": [
+            {
+                "id": "1_1",
+                "video": {
+                    "mimes": [
+                        "video/mp4",
+                        "video/3gpp"
+                    ],
+                    "minduration": 5,
+                    "maxduration": 30,
+                    "protocols": [
+                        7
+                    ],
+                    "w": 1024,
+                    "h": 768,
+                    "startdelay": 0,
+                    "linearity": 1,
+                    "skip": 1,
+                    "skipmin": 5,
+                    "api": [
+                        7
+                    ]
+                },
+                "bidfloor": 0.00123,
+                "ext": {
```

Comment:
how is that `bad imp ext`?
looks just fine, more like `missing imp ext skadn` which is already tested by other adpod tests